src/test/resources/uk/gov/justice/hmpps/prison/api/resource/impl/single_agency_updated_inactive.json

```diff
@@ -0,0 +1,7 @@
+{
+  "agencyId": "LEI",
+  "description": "Leeds",
+  "agencyType": "INST",
+  "active": false,
+  "longDescription": "This is a prison based in Leeds"
```

Comment:
Not sure on this one, but should we return the deactivated date?

Reply:
yes but need to fix the date first